### PR TITLE
0 initial file structure setup

### PR DESCRIPTION
**Pull request by DanielLivshits**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -9,6 +9,11 @@ repositories {
 dependencies {
     implementation 'junit:junit:4.13.1'
     testImplementation('org.junit.jupiter:junit-jupiter:5.6.0')
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.8.1'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.8.1'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.8.1'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.8.1'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.8.1'
```
Comment (Fard-Faru) [inline build.gradle:16]: repeated dependencies

**Review by Fard-Faru — CHANGES_REQUESTED** [review]

**Review by missionStrawberry — APPROVED** [review]
Looks good!

**Review by missionStrawberry — DISMISSED** [review]
Definitely a good idea to remove the redundancy.

File: .gitignore [hunk 2]
```diff
@@ -7,6 +7,7 @@
 .idea/**/usage.statistics.xml
 .idea/**/dictionaries
 .idea/**/shelf
+.idea
```
Comment (Fard-Faru) [inline .gitignore:10]: remove line

File: .gitignore [hunk 3]
```diff
@@ -22,6 +23,7 @@
 .idea/**/dynamic.xml
 .idea/**/uiDesigner.xml
 .idea/**/dbnavigator.xml
+.idea/
```
Comment (Fard-Faru) [inline .gitignore:26]: remove line

File: .gitignore [hunk 4]
```diff
@@ -78,9 +80,11 @@ fabric.properties
 
 ## Gradle-related
 .gradle
+.gradle/wrapper
```
Comment (Fard-Faru) [inline .gitignore:83]: remove line

File: .gitignore [hunk 5]
```diff
@@ -78,9 +80,11 @@ fabric.properties
...
 **/build/
 !src/**/build/
 
+
```
Comment (Fard-Faru) [inline .gitignore:87]: remove line

File: build.gradle [hunk 6]
```diff
@@ -9,6 +9,7 @@ repositories {
 dependencies {
     implementation 'junit:junit:4.13.1'
     testImplementation('org.junit.jupiter:junit-jupiter:5.6.0')
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.8.1'
```
Comment (Fard-Faru) [inline build.gradle:12]: remove line

**Review by POPPININ — COMMENTED** [review]
1. The addition of '.idea' to the .gitignore file makes all other paths with '.idea' as a prefix, redundant.
2. Why are multiple versions of JUnit required in the build.gradle file?

**Review by Fard-Faru — CHANGES_REQUESTED** [review]
Remove Lines which we arent sure is intended for csc207 project format

**Review by Fard-Faru — DISMISSED** [review]
Looks Good

**Review by missionStrawberry — DISMISSED** [review]

**Review by POPPININ — APPROVED** [review]

**Review by Fard-Faru — APPROVED** [review]

**Review by POPPININ — APPROVED** [review]

**Review by missionStrawberry — APPROVED** [review]

**Pull request merged**